ts/modules/overlapping-datalabels.src.ts

```diff
@@ -126,6 +125,17 @@ Chart.prototype.hideOverlappingLabels = function (
         box1,
         box2,
         isLabelAffected = false,
+        isIntersectRect = function (
+            box1: Highcharts.BBoxObject,
```

Comment:
What is `box1` / `box2`, its a little unclear. 

Maybe we should use` box` and `nextBox` or `siblingBox` ?

Reply:
I would prefer to leave it for now as it is - it is an existing method, that I have just re-added. I have tried to move it to utils.js but for now it has no point so needed to move it back in here. 

When this method will go to Utils, the comments and naming may change.